app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
```

Comment:
- [ ] As mentioned in the[ project](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md), please use only users name, to meet the project requirements.